crmd/cib.c

```diff
@@ -68,20 +68,39 @@ do_cib_updated(const char *event, xmlNode * msg)
 
     crm_element_value_int(patchset, "format", &format);
     if (format == 1) {
-        if (get_xpath_object
-            ("//" F_CIB_UPDATE_RESULT "//" XML_TAG_DIFF_ADDED "//" XML_CIB_TAG_CRMCONFIG, msg,
-             LOG_TRACE) != NULL) {
+        if ((get_xpath_object(
+                 "//" F_CIB_UPDATE_RESULT "//" XML_TAG_DIFF_ADDED "//" XML_CIB_TAG_CRMCONFIG,
+                 msg, LOG_TRACE) != NULL) ||
+            (get_xpath_object(
+                 "//" F_CIB_UPDATE_RESULT "//" XML_TAG_DIFF_ADDED "//" XML_CIB_TAG_NOTIFICATIONS,
+                 msg, LOG_TRACE) != NULL)) {
```

Comment:
Beware that get_xpath_object() logs a message and dumps the XML if a match is not found (only at trace level, but that can still be distracting when tracing, if it's not relevant). This probably should have already been using xpath_search() instead (since we don't need the result). Also, wherever possible, we should use a single xpath (xpaths can handle "|" for union, "or" and "and", though I'm not sure of the syntax).